schedule/sle-perf/hana_install.yaml

```diff
@@ -75,6 +75,7 @@ conditional_schedule:
         - installation/partitioning_firstdisk
       ppc64le:
         - installation/partitioning_smalldisk_storageng
+        - '{{fix_sle_15gm_multi_disks}}'
```

Comment:
This schedules `installation/partitioning_smalldisk_storageng` twice. is that right?

Reply:
Thanks for your querying! yes, calling the function twice to workaround different between SLE15 and other product.

Reply:
> This schedules `installation/partitioning_smalldisk_storageng` twice. is that right?

Thanks for your querying! yes, calling the function twice to workaround different between SLE15 and other product.